pkg/api/v1/schema.go

```diff
@@ -758,3 +762,13 @@ type PodNetwork struct {
 	// Default 10.0.2.0/24 if not specified.
 	VMNetworkCIDR string `json:"vmNetworkCIDR,omitempty"`
 }
+
+// Rng represents the random device passed from host
+// ---
+// +k8s:openapi-gen=true
+type Rng struct {
+	// Enabled defines whether use or do not use Virtio Rng
+	Enabled bool `json:"enabled"`
```

Comment:
I wonder whether we need this flag or can we just get away with not setting `Rng`?
If we set it, it becomes enabled otherwise disabled.

Reply:
I think people need to finally realize that minimal spec is useless. That's why we have OS-based presets, and in future even a better solution - virtuned assisted values/defaults.

Vladik's suggestion is cleaner, and makes most sense once we have virtuned.
For the interim we can as well just add explicit rng: to current presets.

Reply:
@michalskrivanek the option is still required for the GUI to have the feedback that something changed and reflect in some user click tool.

However, taking into account your feedback, I propose relaxed version, where you do nothing to opt-in. And explicitly define `disabled: true`, when you do not want RNG added.

@michalskrivanek @senior7515 @vladikr Can you agree on that? The PR is already updated.